roles/ovirt-engine-setup/tasks/main.yml

```diff
@@ -0,0 +1,72 @@
+---
+# health page
+- block:
+  - name: check if ovirt-engine running (health page)
```

Comment:
Why do we check if engine is running there? Do we care? Engine is stopped during engine-setup if running, no?

Reply:
I didn't write this role, but from what I see, we run engine-setup only if the ovirt-engine is not running. Not sure if this is correct. I would even remove this task and the relevant checks.

Reply:
+1 form my side, I think it's expected to execute engine-setup always when you execute this role.

Reply:
If engine is running, engine-setup prompts, asking whether to stop it. If 'no', it aborts.